src/Sylius/Behat/Page/Admin/Product/IndexPageInterface.php

```diff
@@ -25,9 +25,17 @@ public function showProductPage(string $productName): void;
 
     public function chooseChannelFilter(string $channelName): void;
 
+    public function chooseEnabledFilter(): void;
+
     public function filter(): void;
 
+    public function goToPage(int $page): void;
+
     public function checkFirstProductHasDataAttribute(string $attributeName): bool;
 
     public function checkLastProductHasDataAttribute(string $attributeName): bool;
+
+    public function getPageNumber(): int;
```

Comment:
```suggestion
    public function getActivePageNumber(): int;
```
or
```suggestion
    public function getCurrentPageNumber(): int;
```
to give a bit more context